src/main/java/redis/embedded/RedisSentinel.java

```diff
@@ -22,14 +22,20 @@ public RedisSentinel(List<String> args, int port) {
         this.args = new ArrayList<>(args);
     }
 
+    @Override
+    public void setRedisReadyPat(Pattern redisReadyPat) {
```

Comment:
same as before, it should be set in the `Builder` and passed via the class constructor.